units/sys-fs-fuse-connections.mount

```diff
@@ -11,6 +11,7 @@ Documentation=https://www.kernel.org/doc/Documentation/filesystems/fuse.txt
 Documentation=http://www.freedesktop.org/wiki/Software/systemd/APIFileSystems
 DefaultDependencies=no
 ConditionPathExists=/sys/fs/fuse/connections
+ConditionCapability=CAP_SYS_ADMIN
```

Comment:
I boot it in [`docker`](https://github.com/docker/docker/blob/e54ac9b3a1354daf7dd28f9bfd6fc411a25c4531/docs/reference/commandline/run.md#full-container-capabilities---privileged)-mode (`--drop-capability=CAP_SYS_ADMIN`):
```
../../systemd-nspawn \
  --drop-capability=CAP_SYS_ADMIN \
  --register=no \
  --directory=/var/tmp/systemd-test.JkOcwx/nspawn-root \
/usr/lib/systemd/systemd systemd.log_level=debug systemd.unit=rescue.target
```